pkg/cosign/verify.go

```diff
@@ -135,6 +135,16 @@ func VerifyTLogEntry(rekorClient *client.Rekor, uuid string) (*models.LogEntryAn
 	if err := v.VerifyInclusionProof(*e.Verification.InclusionProof.LogIndex, *e.Verification.InclusionProof.TreeSize, hashes, rootHash, leafHash); err != nil {
 		return nil, errors.Wrap(err, "verifying inclusion proof")
 	}
+
+	// Verify rekor's signature over the SET.
+	rekorPubKey, err := PemToECDSAKey([]byte(rekorPub))
```

Comment:
should i just get this from rekorClient instead of using the offline one?

Reply:
personally i think this is fine

Reply:
agreed! i actually might change it over so we don't have to use the prod instance in the e2e test (which is why it's failing)